### PR TITLE
[Feature] adding prettier and update eslint and some files

### PR DESCRIPTION
**Pull request by ramonxm**
## Description

Adicionando prettier e atualizando eslint para manter um padrão no projeto.

## Developer Checks

- [x] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [x] Tested on browser device
- [x] Tested on Mobile device

### CODE REVIEW

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```
Comment (jonataspinto) [inline .prettierrc:2]: Só uma curiosidade, nossa preferencia por duplas está relacionada com auto import que sempre coloca duplas.

**Review by jonataspinto — APPROVED** [review]

**Comment by ramonxm** [comment]
[Implementar Eslint, Prettier e Stylelint.](https://trello.com/c/vctaKl1w/22-implementar-eslint-prettier-e-stylelint)

File: .prettierrc (hunk 1 above)
Comment (ramonxm) [reply .prettierrc:2]: Se você tiver o prettier instalado no vscode, a regra do prettierrc prevalece e o auto import vem com single quote

**Pull request merged**